### PR TITLE
Benchmark support for HiKey6220 and TI SoCs(am57xx, dra7xx)

### PR DESCRIPTION
**Pull request by igoropaniuk**
To be able to run benchmarks on TI platforms "OP-TEE Benchmark" patch         
for linux optee driver should be applied manually (as they are using their own linux fork), `CONFIG_OPTEE_BENCHMARK=y` flag should be added to `ti_config_fragments/baseport.cfg`.

### CODE REVIEW

File: common.mk [hunk 1]
```diff
@@ -24,6 +24,9 @@ LIBYAML_LIB_PATH		?= $(BENCHMARK_APP_PATH)/libyaml/out/lib
 CFG_TEE_CORE_LOG_LEVEL		?= 3
 
 # default disable latency benchmarks (over all OP-TEE layers)
+# To be able to run benchmarks on TI platforms "OP-TEE Benchmark" patch
```
Comment (jbech-linaro) [inline common.mk]: Sentence is a bit hard to follow, do you mean something like:
`# To be able to run benchmarks on TI platforms, the "OP-TEE Benchmark" patch(es) in Linux kernel needs to be applied manually by setting CONFIG_OPTEE_BENCHMARK=y`

Comment (igoropaniuk) [reply common.mk]: Actually I meant that two steps should be done:
1) Cherry-pick OP-TEE benchmark patches from our linaro-swg/linux rep, and apply it (as in TI manifests their own linux fork is used)
2) Manually add `CONFIG_OPTEE_BENCHMARK=y` to `ti_config_fragments/baseport.cfg.`

Comment (jbech-linaro) [reply common.mk]: > Cherry-pick OP-TEE benchmark patches from our linaro-swg/linux rep, and apply it (as in TI manifests their own linux fork is used)

I think we instead should put the benchmark patches on the LSK branch that TI are using 
`branch: v4.9/topic/optee (git://git.linaro.org/kernel/linux-linaro-stable.git)`.

Comment (igoropaniuk) [reply common.mk]: @etienne-lms 
So I guess I should take a break for changes that enable benchmarks on TI SoCs(remove this commit), so we can just add support for HiKey620. What do you think?

And when benchmark patches will be upstreamed to LSK branch, then we will return to TI benchmark enabling patches here in `built.git`

Comment (jbech-linaro) [reply common.mk]: Igor, let's sync up on this on our next 1:1.

Comment (igoropaniuk) [reply common.mk]: @jbech-linaro sure, will add it to 1:1 doc.
